src/Numerical/Spectral/MeanValue.hpp

```diff
@@ -0,0 +1,41 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+/// \file
+/// Defines function MeanValue.
+
+#pragma once
+
+#include "Domain/Side.hpp"
+#include "Numerical/Spectral/DefiniteIntegral.hpp"
+#include "Utilities/ConstantExpressions.hpp"
+
+/*! \ingroup Functors
+ * \brief Compute the mean value of a grid-function over a manifold.
+ *
+ * \remarks The mean value is computed on the reference element(s).
```

Comment:
What does this mean?

Reply:
This means the function computes `mean = \int f dV / \int dV` where dV is a volume element of the reference coordinates (dxi,deta,dzeta). In a different coordinate system x', dV' is a complicated function, giving a different mean.

I like the remark as currently phrased, but a possible alternative phrasing would be `The mean value is computed w.r.t. the reference coordinates`.

I think it would be nice to add a note indicating how to compute the mean w.r.t. other coordinates, something like: `The mean w.r.t. a different set of coordinates x can be computed by pre-multiplying the argument f by the Jacobian J = dx/dxi of the mapping from the reference coordinates xi to the coordinates x`.

Reply:
ping